src/Symfony/Component/DependencyInjection/composer.json

```diff
@@ -29,6 +29,7 @@
     "suggest": {
         "symfony/yaml": "",
         "symfony/config": "",
+        "symfony/expression-language": "For using expression matching in service container configuration files",
```

Comment:
It was an unsuitable reuse of an [existing one](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Routing/composer.json#L37), thank you.
The change has been made.